app.js

```diff
@@ -13,6 +13,24 @@ const { json, urlencoded } = express;
 //Routes
 
 var app = express();
+const http = require('http').createServer(app);
+const io = require('socket.io')(http,  {
+  handlePreflightRequest: (req, res) => {
+      const headers = {
+          "Access-Control-Allow-Headers": "Content-Type, Authorization",
+          "Access-Control-Allow-Origin": req.headers.origin, //or the specific origin you want to give access to,
+          "Access-Control-Allow-Credentials": true
+      };
+      res.writeHead(200, headers);
+      res.end();
+  },
+  cors: {
+    origin: "https://example.com",
+    methods: ["GET", "POST"],
+    allowedHeaders: ["my-custom-header"],
```

Comment:
Looks like some of the socket stuff still wound up here. Let's be sure to remove the template stuff!

Reply:
Interesting, they seem to have disappeared in my end, and i deleted my branch